spec/lcm/load/lcm_load_spec.rb

```diff
@@ -12,6 +12,7 @@
 $master_projects = []
 $client_projects = []
 $master = false
+$start_time
```

Comment:
Lint/Void: Variable `$start_time` used in void context.

Reply:
what does this do?